pages/index.tsx

```diff
@@ -170,6 +170,15 @@ const code = `ReactPPTX.render(
     <Slide>
       <Table
         rows={[
+          [<Table.Cell 
+            colSpan={2} 
+            style={{ 
+              align: "center", 
+              backgroundColor: '#115599', 
+              color: 'white'
+            }}>
+            Title
+          </Table.Cell>],
           ["foo", <Text style={{ align: "right", backgroundColor: '#404040' }}>bar</Text>],
```

Comment:
I guess all `<Text>` elements used in `rows` could be made into `<Table.Cell>` now to be a bit more consistent?

Reply:
Done ✅